index.html

```diff
@@ -0,0 +1,228 @@
+        <div class="main-program">
+          <p>Main Program</p>
+        </div>
+
+        <div class="border"></div>
+      
+        <div class="insight-section">
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_01.png" alt="lecture-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Lecture
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_02.png" alt="exhibition-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  DMT Exhibition
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Appreciate various creations applying DMT, organized from football Center Akwa Ibom.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_03.png" alt="forum-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Forum
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Have the time to share your thoughts and opinions with expert for each topic.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_04.png" alt="workshop-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Workshop
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Try creating your own work using open source license rather than just watching.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_05.png" alt="ignite-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  DMT Ignite
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Get opportunities to network with DMT affiliate around the world, also after the summit.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="base-text">
+          <p>
+            Join DMT Global Summit 2019
+          </p>
+        </div>
```

Comment:
- [ ] Kindly use a `button` tag or `a` tag for this element. Using a button tag makes it more semantic because ideally, this should direct you to a page to join the event. 👍 💻

Reply:
Done